src/desktop/apps/auction_reaction/server.tsx

```diff
@@ -3,4 +3,5 @@ import { bidderRegistration } from "./routes"
 
 export const app = express()
 
+app.get("/auction-registration/:auctionID", bidderRegistration)
```

Comment:
Does this handle the request first just because `auction_reaction` comes before `auction_support` alphabetically?

Reply:
It is also loaded first in the [express app setup](https://github.com/artsy/force/blob/b05c38dd60581a1df58ed699b2b6b1615a96914d/src/desktop/index.js#L23-L30) so in a pattern matching sense this route loads first.